src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -39,6 +40,9 @@ export const isIntegrationDirectoryActive = (organization?: Organization) => {
   }
 };
 
+export const isSortIntegrationsByWeightActive = () =>
```

Comment:
@leedongwei But, the function returns a boolean value.

Reply:
@leedongwei what name would you suggest instead?

Reply:
I would prefix with `get` 
1. getters and setters denotes functions
2. this is a transformation of the data in localStorage because localStorage can only store strings

But it's a 1-line function and I would not abstract this.

You could make an argument that it abstracts the value of `'1'` being used as `True`, but if that's a real concern, you should declare that as a constant.

Reply:
@leedongwei this function just checks for value in `localStorage`. It's just a placeholder for the next ticket where we will be adding more complicated logic similar to `isIntegrationDirectoryActive` on line 31 of this file.

Reply:
Then it should be prefixed with `get`.

Now that I looked at `isIntegrationDirectoryActive`, IMO it should be rewritten too.

This is not great
```
if (something) {
  return something
} else if (anotherthing) {
  return anotherthing
} else {
  return thisthing
}
```

This is good because it improves readability and discourages nested if-statements
```
if (something) {
  return something
} 

if (anotherthing) {
  return anotherthing
} 

return thisthing
}
```

Reply:
@leedongwei This function is a 1 liner now but it is going to look like `isIntegrationDirectoryActive` once we add the A/B test. So the abstraction makes sense now. Definitely agree on the structure of `if` and `else` statements. We are going to remove `isIntegrationDirectoryActive` soon so it doesn't make sense to update that but we should definitely follow your proposed structure in `isSortIntegrationsByWeightActive` (or whatever the new name is going to be).